### ISSUE TITLE
Please output INFO messages in DefaultBuildContext.addMessage

### ISSUE THREAD

**Issue by rakus** [codehaus-plexus/plexus-build-api#20]
Thank you very much for reviving this project. It was strange to have so many components that depend on a module that was archived.

Please consider to expand the method `DefaultBuildContext.addMessage(...)` to also log info messages. Currently only ERROR and WARNING are printed. This would mean adding a new constant `SEVERITY_INFO` to the interface `BuildContext` and extending the 
the switch statement in the `addMessage(...)` implementation. Maybe also a default branch would be useful.

Note that adding a INFO message (severity == 0) inside Eclipse works and adds a INFO IMarker to the file (tested with 2021-12).

**Comment by rakus** [comment]
Just realized: Adding a constant to the interface `BuildContext` would break binary compatibility wit 007. Not good.
Anyway `addMessage(....)` could handle severity == 0 as INFO.

**Comment by slachiewicz** [comment]
Worth to add to 1.0.0 @olamy

**Comment by laeubi** [comment]
Good idea! Why not make it 1.0.0 **now**? also change the package name and then all its fine, I don't think binary compatibility with 007 is anything useful:

1. There are mostly consumers of the API so this won't break for them
2. Don't know if there are other implmentors beside m2e and as we need to possibly adapt here anyways it won't be an issue at all
3. Why would adding a constant break compatibility anyways?
